Krypt/Source/SHA256.swift

```diff
@@ -20,4 +20,41 @@ public struct SHA256 {
     }
     return status == errSecSuccess ? Data(hash) : nil
   }
+
+  // MARK: - Buffered SHA-256 Calculation
+
+  public static func digest(from file: URL, with bufferSize: Int = 1024 * 1024) -> Data? {
```

Comment:
```suggestion
  public static func digest(file url: URL, withBufferSize bufferSize: Int = 1024 * 1024) throws -> Data {
```
Makes it clearer at call site: `SHA256.digest(file: URL(...), withBufferSize: 1_000_000)`

And with throwing it's better in case we need to handle specific error cases, especially since `FileHandle` is used.